android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java

```diff
@@ -809,6 +809,10 @@ public int getLineCount()
 	public String getVisibleText()
 	{
 		MaterialTextView textView = (MaterialTextView) getNativeView();
-		return textView.getLayout().getText().toString();
+		if (textView != null && textView.getLayout() != null) {
```

Comment:
This looks unrelated to this change?

Reply:
I believe @m1ga found one test failing because of missing null check here.

Reply:
@m1ga `null` checks are missing in above `getLineCount` method and at some other places as well in this class.

Reply:
yes, one test was failing:

> one test was throwing a "getText() of null" error for the label so I've added a null check

so I've added the fix here. The `getLineCount()` method doesn't have a test but I think you are right it might fail too. We have those tests in [other methods too](https://github.com/tidev/titanium-sdk/blob/4532539fd9cd538f123889d8ccddaf535b42e0f1/android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java#L715-L718). I'll update it later.

Reply:
I've split it into 2 PRs so the code change is here now: https://github.com/tidev/titanium-sdk/pull/14174